### PR TITLE
Docs_devel: document, L10N, website and scripting

### PR DESCRIPTION
**Pull request by miurahr**
Update contribution guide

## Pull request type

- Documentation -> [documentation]

## Which ticket is resolved?

Dev list discussion

https://sourceforge.net/p/omegat/mailman/message/37755583/

## What does this PR change?

- Add section for documentation and localization contribution
- Put place holder for Scripting contribution guide
-

### CODE REVIEW

**Comment by brandelune** [comment]
I think @kosivantsov will be able to mostly copy-paste the contents from the L10N page _or_ make a summary and link to it. 

Philippe and I will try to write something about documentation authoring. But the process needs to be discussed together, since it really has to be integrated with development so that we don't have discrepancies between the code and the doc.

**Review by brandelune — COMMENTED** [review]
@Kazephil will probably change a few things but I'm fine with the general contents.

File: docs_devel/docs/00.ContributingEcoSystem.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+
+### Plugins
+
+OmegaT has a mechanism to add extension plugins. When you want to introduce a new feature that make OmegaT fitting your 
+needs and believe it helps others, plugins is a first candidate to publish your feature.
+
+There are many plugins in several categories such as
+
+- Theme
```
Comment (damien-rembert) [inline docs_devel/docs/00.ContributingEcoSystem.md:79]: Themes would be better as the other items are also plural

**Review by damien-rembert — APPROVED** [review]
It all looks good to me.
Maybe I'm a bit lyrical today but I think it would be nice to add a bit of symmetry to the different types of Contribution, maybe something like "Contributing Code", "Contributing Documentation" and "Contributing Translation"?

**Comment by Kazephil** [comment]
Give me another day or two to read through the material.
I'll incorporate the suggestions from @damien-rembert and get everyone to look over any other changes I make, then push the final version to SourceForge.

**Comment by brandelune** [comment]
Thank you all for working on this document.

**Comment by miurahr** [comment]
@Kazephil thank you for cooperation. Could you please update on another topic branch of you for your improvement.

**Pull request merged**